docs/Build/node-cli.md

```diff
@@ -71,8 +71,15 @@ Restart=on-failure
 RestartSec=5
 User=$USER
 LimitNOFILE=1048576
-WorkingDirectory=/opt/cardano/cnode/scripts
-ExecStart=/bin/bash -l -c 'exec \"\$@\"' _ cnode.sh
+Environment=CNODE_HOME=/opt/cardano/cnode
+WorkingDirectory=/home/cardano/.cabal/bin
+ExecStart=cardano-node run \\
```

Comment:
Also - an after thought - this might start creating relay nodes instead of Pools from peepz , so can become a bit confusing  , where peepz modify cnode.sh and not systemd.

Question of whether we spoon feed them or we just use bash -l -c exec ... perhaps

Reply:
If they follow the guide, cardano-node should be located in /home/cardano/.cabal/bin and as such setting WorkingDirectory to this should be enough. If a custom installation is performed, end-user is should be able to adapt instructions for their needs. 

But I do agree with the statement that we need to handle cnode.sh vs systemd better. Ie make it more clear what changes are needed and to what files when upgrading a passive node to a block producing core node.

Can we assume systemd is available on all systems and just drop the cnode.sh script or should we keep both going forward?